### PR TITLE
@yuki24: Gallery Partnership signup page QA items

### PR DESCRIPTION
**Pull request by xtina-starr**
This addresses the QA items mentioned in https://github.com/artsy/collector-experience/issues/94#issuecomment-313482097

It might be worth while to pull down this branch and review the issues mentioned in QA if you have a chance.

### CODE REVIEW

File: desktop/apps/gallery_partnerships2/client/index.coffee [hunk 1]
```diff
@@ -5,8 +5,8 @@ module.exports.init = ->
   if sd.USER_AGENT.match('iPhone')
     $('.gallery-partnerships2__chat__video').get(0).play()
 
-  initCarousel $('.js-partner-stats-slideshow'), { autoPlay: 2500, wrapAround: true }
-  initCarousel $('.js-partner-testimonials-slideshow'), { autoPlay: 2500, wrapAround: true, prevNextButtons: false }
+  initCarousel $('.js-partner-stats-slideshow'), { autoPlay: 2500, wrapAround: true, draggable: false }
```
Comment (xtina-starr) [inline desktop/apps/gallery_partnerships2/client/index.coffee]: This disables the partner slide show carousel from being draggable.

File: desktop/apps/gallery_partnerships2/stylesheets/index.styl [hunk 2]
```diff
@@ -33,6 +33,12 @@
   @media screen and (max-width: 768px)
     flex-wrap wrap-reverse
 
+.max-width-on-mobile
```
Comment (xtina-starr) [inline desktop/apps/gallery_partnerships2/stylesheets/index.styl]: I added this helper class in order to set the max-width on some elements on the smaller screens breakpoint. This is related to the first item Daniel mentioned in his QA feedback.

File: desktop/apps/gallery_partnerships2/stylesheets/mobile.styl [hunk 3]
```diff
@@ -1,62 +0,0 @@
-@media screen and (max-width: 768px)
```
Comment (xtina-starr) [inline desktop/apps/gallery_partnerships2/stylesheets/mobile.styl]: we added the style rules for mobile in the `index.styl` file so removing this.

**Comment by yuki24** [comment]
I have tested this locally on Safari (Mac & iOS), Chrome (Desktop & Android), Firefox and IE (11 & Edge) and IE11 is the only one that doesn're render flexbox.

screen shot 2017-07-14 at 4 19 04 pm

I'm looking into this now and will send a fix once I figure it out.

**Comment by yuki24** [comment]
It seems like we are hitting [this bug](https://connect.microsoft.com/IE/feedback/details/802625/min-height-and-flexbox-flex-direction-column-dont-work-together-in-ie-10-11-preview) and

screen shot 2017-07-14 at 4 56 48 pm

**Comment by yuki24** [comment]
Looks great @xtina-starr! merging this...

**Comment by xtina-starr** [comment]
😄 thanks for adding the IE fix and merging @yuki24!!

**Pull request merged**